### DISCUSSION TITLE
Questions that we'd like to ask from DocQuery

### DISCUSSION THREAD

**Discussion by ankrgyl** [impira/docquery#35] [Ideas]
This discussion helps track a "dream list" of questions we'd like to be able to ask via DocQuery. They can and should expand beyond the scope of a single document.

**Comment by impirawil** [comment]
"redact assignee name"
"highlight assignee name" 
"erase assignee name" 
"tell me the assignee's name(s)"
"copy the assignee's name(s)"
"make a note of the assignee's name(s)" 
"rename all contracts based on assignee name" 
"move all contracts where the assignor is ABC Company to a new folder"
"are all of the signatures added that are required?"

**Comment by impirawil** [comment]
"delete all contracts where assignee name is empty"
"delete incomplete forms"
with a watcher: "send completed forms to such and such"

**Comment by impirawil** [comment]
"organize into separate folders based on contractor name"
"move all invoices where the ship to and bill to are the same to x folder"
"zip all files where the vendor name is Company name"
"create spreadsheet for all invoices with a different tab for each vendor"

**Comment by impirawil** [comment]
"summarize all invoices with account name, number, and total due"
"add up all of the totals"
"email each assignee a copy of the document where they appear"

**Comment by ankrgyl** [comment]
From [Twitter](https://twitter.com/dbarber/status/1573287231758143488)

> The 401k documents have standards that list fees.The challenge can be that most people have multiple funds in their portfolio (multiple documents).It would be useful to summarize fees for a portfolio (fees can be buried in prospectus) eg. https://www.capitalgroup.com/individual/investments/fund/amcpx#detail-page-dig-anchor

**Reply by TestDataGuy** [reply]
A more practical use for Docquery would be pulling data from brokerage monthly account statements (Schwab & Fidelity) to reconcile values in internal portfolio accounting software. 

"Show me accounts that have different values for securities in their brokerage statement and our quarterly reports"

**Reply by ankrgyl** [reply]
Awesome thank you!!

**Comment by ankrgyl** [comment]
From [LinkedIn](https://www.linkedin.com/feed/update/urn:li:activity:6971204053871599616/?commentUrn=urn%3Ali%3Acomment%3A(activity%3A6971204053871599616%2C6979932233918812160)&dashCommentUrn=urn%3Ali%3Afsd_comment%3A(6979932233918812160%2Curn%3Ali%3Aactivity%3A6971204053871599616))

> Show me all the invoices from 09-01 to 09-23 that were issued to Acme*

**Comment by impirawil** [comment]
"list the top earners and their 3 highest contracts by region for the years 2018-2022"
"who is the highest paying contractor?"
"create an interactive pie chart that shows the monthly spend for each vendor"
"create a contact list for all unique payers"
"how many people are past due by over a month. do _blank_ with a list of any who owe more than $1000.000."
